cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go

```diff
@@ -160,5 +167,10 @@ func IsCloudProviderNodeInfoSimilar(n1, n2 *schedulerframework.NodeInfo, ignored
 		return false
 	}
 
+	fmt.Println(includedLabels, len(includedLabels))
+	if len(includedLabels) > 0 && !compareLabels(nodes, includedLabels) {
```

Comment:
Wasn't the #3615 intent to shortcut the full labels comparison (the `if !compareLabels(nodes, ignoredLabels)`  above) with a comparison over a smaller set of `includedLabels`?

Reply:
You are correct @bpineau . I will fix the PR